src/Behat/Testwork/Tester/ServiceContainer/TesterExtension.php

```diff
@@ -159,14 +181,21 @@ protected function loadStrictController(ContainerBuilder $container, $strict = f
      *
      * @param ContainerBuilder $container
      */
-    protected function loadResultInterpreter(ContainerBuilder $container)
+    protected function loadResultInterpreter(ContainerBuilder $container, bool $strict)
     {
         $definition = new Definition('Behat\Testwork\Tester\Result\ResultInterpreter');
         $container->setDefinition(self::RESULT_INTERPRETER_ID, $definition);
 
         $definition = new Definition('Behat\Testwork\Tester\Result\Interpretation\SoftInterpretation');
         $definition->addTag(self::RESULT_INTERPRETATION_TAG);
         $container->setDefinition(self::RESULT_INTERPRETATION_TAG . '.soft', $definition);
+
+        
+        if ($strict) {
+            $definition = new Definition('Behat\Testwork\Tester\Result\Interpretation\StrictInterpretation');
+            $definition->addTag(TesterExtension::RESULT_INTERPRETATION_TAG);
```

Comment:
service with RESULT_INTERPRETATION_TAG are automatically registered in the result interpreter, so we do not need the StrictHandler anymore